syft/formats/syftjson/to_format_model.go

```diff
@@ -17,8 +17,6 @@ import (
 )
 
 // ToFormatModel transforms the sbom import a format-specific model.
-// note: this is needed for anchore import functionality
-// TODO: unexport this when/if anchore import functionality is removed
```

Comment:
This is still needed to be exported since the`template` package is using it in its `Encode` function